presentation/src/main/java/com/shifthackz/aisdv1/presentation/activity/AiStableDiffusionViewModel.kt

```diff
@@ -27,6 +29,9 @@ class AiStableDiffusionViewModel(
 
     override val effectDispatcher = dispatchersProvider.immediate
 
+    private val _isShowSplash = MutableStateFlow(true)
+    val isShowSplash = _isShowSplash.asStateFlow()
```

Comment:
Why introduce another stateflow variable in MVI viewmodel?
Please add this as a variable to the MviState of this viewmodel.

Reply:
Resolved.